### PR TITLE
[EDA] Reciprocal Relationship Mapping Recommended Fix Updates

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -2036,15 +2036,15 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Recommended fix when the reciprocal setting&apos;s Name value is inactive</shortDescription>
-        <value>Verify the reciprocal setting for {0} ({1}) specified in EDA Settings, on the Reciprocal Settings subtab of the Relationships tab. If {0} ({1}) is correct, go to the Relationship object&apos;s Type field in Salesforce Setup and set the picklist value to Active. If it isn&apos;t correct, change the reciprocal setting&apos;s Name value to a valid, active picklist value.</value>
+        <value>The Reciprocal Relationships mapping for {0} ({1}) isn&apos;t active. In EDA Settings | Relationships, verify the mapping that specifies this Name. If the mapping is correct, go to the Relationship object&apos;s Type field in Salesforce Setup and set the picklist value to Active. If it isn&apos;t correct, change the reciprocal setting&apos;s Name value to a valid, active picklist value.</value>
```
Comment (xinnanlisf) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:2039]: The label in the spec doc does not have ({1}). Just fyi

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 2]
```diff
@@ -2060,7 +2060,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Recommended Fix when Name isn&apos;t unique in reciprocal settings</shortDescription>
```
Comment (xinnanlisf) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:2062]: The short description here isnt matching the spec.

**Review by xinnanlisf — COMMENTED** [review]
@skosgi1 overall looks great. Left a few comments.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (skosgi1) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:2039]: @xinnanlisf I think that is a typo in the SPEC, I fixed it now.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 2 above)
Comment (skosgi1) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml]: Fixed it, thanks for catching this.

File: force-app/main/default/translations/ca.translation-meta.xml [hunk 3]
```diff
@@ -997,39 +997,39 @@
         <name>stgHCReciprocalRelFemaleInactiveFix</name>
```
Comment (xinnanlisf) [inline force-app/main/default/translations/ca.translation-meta.xml:997]: Assuming we need to reset the translation for this label too?

Comment (skosgi1) [reply force-app/main/default/translations/ca.translation-meta.xml:997]: Done

**Review by xinnanlisf — APPROVED** [review]

**Comment by spinkelman** [comment]
QA passes - Merging

**Pull request merged**